virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -18,19 +18,31 @@ To attach a virtual machine to an additional network:
 The network attachment definition must be in the same namespace as the pod or virtual machine.
 ====
 . Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
+. Configure a Linux bridge.
```

Comment:
How to configure a bridge?
I think that a link to the page that explains that (https://deploy-preview-43988--osdocs.netlify.app/openshift-enterprise/latest/virt/node_network/virt-updating-node-network-config.html) may be useful.